plugin/vscode/quick-lint-js.js

```diff
@@ -160,9 +162,14 @@ class Parser {
     }
   }
 
-  lint() {
+  lint(performanceWriter) {
+    let lint_start;
+    let lint_end;
```

Comment:
Nit: Variable names should be `lowerCamelCase`:
```suggestion
    let lintStart;
    let lintEnd;
```

Reply:
Used your suggested Changes in latest commit